### ISSUE TITLE
Example cifar10_serial.py returns zero loss and train accuracy after each epoch (main branch)

### ISSUE THREAD

**Issue by chariako** [APPFL/APPFL#189]
In the examples folder of main APPFL branch, running the command 

>  python ./cifar10_serial.py --num_clients 2 --partition iid --loss_fn losses/celoss.py --loss_fn_name CELoss --num_epochs 10 --train_data_batch_size 8

returns zero loss and training accuracy after each epoch.

**Comment by Zilinghan** [comment]
Hi @chariako and @kibaekkim, the reason you are seeing this is because you are not enabling validation, so by default, it shows 0. To see the accuracy, run the following - including the `--do_validation` flag
`python ./cifar10_serial.py --num_clients 2 --partition iid --loss_fn losses/celoss.py --loss_fn_name CELoss --num_epochs 10 --train_data_batch_size 8 --test_data_batch_size 8 --do_validation`

**Comment by Zilinghan** [comment]
Btw, the validation process could be a bit slow if you are running on CPU, few free to change [this line](https://github.com/APPFL/APPFL/blob/f36dd80cad5cf6e268bd798bfbd3b4db9f9fcaad/examples/dataloader/cifar10_dataloader.py#L40) from `test_dataset = Dataset(torch.FloatTensor(test_data_input), torch.tensor(test_data_label))` to `test_dataset = Dataset(torch.FloatTensor(test_data_input[:20]), torch.tensor(test_data_label[:20]))` to make the test set a bit smaller.

**Comment by chariako** [comment]
Thanks @Zilinghan, I can see the loss and accuracy values after each epoch if I add the --do_validation flag. I will mark this issue as closed.
